scripts/build.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
 set -e
 
-npm install -g cross-env
```

Comment:
Why was this removed?

Reply:
Wasn't necessary to install globally according to @coderasher as its a library.

Reply:
Yup it's part of the root `package.json`.